### PR TITLE
!Fix actionbar dropdown clipping and ie width

### PR DESCRIPTION
**Pull request by frederik-labbe**
[ADUI-352]

### CODE REVIEW

**Comment by marilynechan** [comment]
Don't forget to run `gulp sass:format` before submitting your PR, that's why the build failed.
Ship it!

File: scss/components/application-container.scss [hunk 1]
```diff
@@ -2,6 +2,6 @@
   margin-top: $header-height;
   margin-left: $navigation-width;
   overflow: hidden;
-
+  min-height: 600px;
```
Comment (pochretien) [inline scss/components/application-container.scss]: I think we can add this 600px in a variable because we use it for the application container

**Comment by pochretien** [comment]
sass format and :ship:  it

**Comment by wfortin** [comment]
:+1:

**Pull request merged**